### DISCUSSION TITLE
Ability to transfer data and schema from one db to another.

### DISCUSSION THREAD

**Discussion by dotsinspace** [prisma/prisma#23271] [Q&A] [answered]
### Question

I didn't find out any reliable solution for transffering data from one DB to another. now this could be done using scripts ( either in python or js both of them have Prisma support ) but i want something reliable like db push for schema.

This could help team to build db of same type of data more efficiently. use case.

i have DB of billion entries of Networks..now have to transfer those entries to another db.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @dotsinspace 👋 

Thank you for raising this question.

Prisma's `db push` is primarily used for quickly prototyping and iterating on schema design locally.  It does not address the use case of transferring large volumes of data between databases. While Prisma can help manage your database schema and interact with your database through Prisma Client, the specific use case of transferring large volumes of data from one database to another typically falls outside the direct use case Prisma is designed for. 

For transferring data between databases, especially with the scale you're describing (billions of entries), a custom script might still be the most flexible and controlled approach. This could involve exporting the data from the source database and importing it into the target database, potentially using intermediate formats like CSV or JSON, depending on the databases involved and their support for import/export formats.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
